cmd/gb-vendor/update.go

```diff
@@ -59,45 +59,38 @@ Flags:
 		}
 
 		for _, d := range dependencies {
-			url := d.Repository
-			p := d.Importpath
-
 			err = m.RemoveDependency(d)
 			if err != nil {
 				return fmt.Errorf("dependency could not be deleted from manifest: %v", err)
 			}
 
-			if err := os.RemoveAll(filepath.Join(ctx.Projectdir(), "vendor", "src", filepath.FromSlash(p))); err != nil {
+			if err := os.RemoveAll(filepath.Join(ctx.Projectdir(), "vendor", "src", filepath.FromSlash(d.Importpath))); err != nil {
```

Comment:
@davecheney What do you think about this long line? Wouldn't it be more readable if it was split into multiple lines?